src/_cffi_src/build_openssl.py

```diff
@@ -37,7 +37,11 @@ def _osx_libraries(build_static):
         return ["ssl", "crypto"]
 
 
-_OSX_PRE_INCLUDE = """
+_PRE_INCLUDE = """
+#include <openssl/e_os2.h>
```

Comment:
Does this need to be after the OS X crap to avoid a warning, or does it work out ok?